zubhub_frontend/zubhub/src/views/PageWrapper.jsx

```diff
@@ -103,15 +108,16 @@ function PageWrapper(props) {
     }
   };
 
+
   const { anchor_el, loading, open_search_form } = state;
   const { t } = props;
   const { zubhub } = props.projects;
-  const profileMenuOpen = Boolean(anchor_el);
+  const profileMenuOpen = Boolean(anchor_el) 
```

Comment:
you removed `;` from this line. maybe add it back? it doesn't hurt (yet) to leave it out, but for the purpose of convection and avoiding future hard-to-track bugs, it makes sense to put it back.

Reply:
ah ok i understand i will put it again